src/createRecords.ts

```diff
@@ -0,0 +1,64 @@
+import AirtableSDK from "airtable"
+
+import { SchemaZodT } from "./types/schema"
+import { SelectQueryParamsT } from "./types/queryParams"
+
+export interface CreateRecords<T extends SchemaZodT> {
+	tableId: string
+	baseId: string
+	schema: T
+	query?: SelectQueryParamsT
+	records: Array<T>
+}
+
+export interface CreateRecordsWithAirtable<T extends SchemaZodT>
+	extends CreateRecords<T> {
+	airtable: AirtableSDK
+}
+export interface CreateRecordsWithApiKey<T extends SchemaZodT>
+	extends CreateRecords<T> {
+	apiKey: string
+}
+
+//TODO should we change this to an upsert operation?
```

Comment:
@iainkirkpatrick @hikurangi whats your opinion on just having upsert rather then Create and Update functions?